doc/advanced-topics.md

```diff
@@ -73,15 +72,18 @@ Agent.
 
 Complete info on Keystone trust tokens could be found at:
 
-- [Trusts concept](https://docs.openstack.org/keystone/stein/user/trusts)
-- [Trusts API](https://docs.openstack.org/keystone/stein/api_curl_examples.html#post-v3-os-trust-trusts)
+-   [Trusts concept](https://docs.openstack.org/keystone/stein/user/trusts)
+-   [Trusts API](https://docs.openstack.org/keystone/stein/api_curl_examples.html#post-v3-os-trust-trusts)
+
+### NGSI-LD `GeoProperty` support
```

Comment:
In the ToC at the beginning of the file I see:

```
[GeoJSON support NGSI-LD only](#geojson-support-ngsi-ld-only)
```

So it seems section title and corresponding entry in ToC are not aligned...

Reply:
Fixed ef3dd11